phasespace/phasespace.py

```diff
@@ -706,7 +706,8 @@ def generate_tensor(
 
         Raise:
             tf.errors.InvalidArgumentError: If the the decay is kinematically forbidden.
-            ValueError: If `n_events` and the size of `boost_to` don't match. See `GenParticle.generate_unnormalized`.
+            ValueError: If `n_events` and the size of `boost_to` don't match.
+            See `GenParticle.generate_unnormalized`.
```

Comment:
Why is this on the next line?

Reply:
flake was complaining about the line width (perhaps a good example of a false positive :grimacing:). Sadly, docformatter didn't pick that up (maybe because of the backticks?)